pulp_python/app/models.py

```diff
@@ -34,6 +34,45 @@ class Classifier(Model):
                                                on_delete=models.CASCADE)
 
 
+class DistributionDigest(Model):
+    """
+    A model of digests on an individual distribution.
+    """
+    type = models.TextField()
+    digest = models.TextField()
+    project_specifier = models.ForeignKey("ProjectSpecifier",
+                                          related_name="digests",
+                                          related_query_name="distributiondigest",
+                                          on_delete=models.CASCADE)
+
+
+class ProjectSpecifier(Model):
+    """
+    A specifier of a python project dependency.
+
+    Example:
+
+        version==1.0.0 will sync all distributions matching version
+        version~=1.0.0 will sync all major version 1 distributions
+        version==1.0.0 digests=["sha256:0000"] will only sync the distributions matching the hash
+        name=projectname without specifiers will sync every distribution in the project.
```

Comment:
Does digests work separately from a version specifier?  If yes, could you provide an example for just digests?

Could you provide another example using  ```>=0.9, <1.0```?

Reply:
^

Reply:
@dralley Ah I thought you meant add an example to the workflow docs. I'll add those to the docstring too